OxfordBodleian/Ullendorff/BDLeu9.xml

```diff
@@ -45,48 +121,132 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+                                    <dimensions type="margin" unit="mm" xml:id="margin1">
+                                        <dim type="right">5–15</dim>
+                                        <dim type="left">5–15</dim>
+                                    </dimensions>
+                                    <ab type="pricking">Pricking and ruling are not visible.</ab>
+                                </layout>
+                                
+                                <layout columns="1" writtenLines="51">
+                                    <note>Number of characters per line: 9–10.</note>
+                                    <note>The outer margins are delimited by a black line.</note>
+                                    <dimensions unit="mm" xml:id="textarea2">
+                                        <height>413</height>
+                                        <width>55–65</width>
+                                    </dimensions>
+                                    <dimensions type="margin" unit="mm" xml:id="margin2">
+                                        <dim type="right">5–15</dim>
+                                        <dim type="left">5–15</dim>
+                                    </dimensions>
+                                    <ab type="pricking">Pricking and ruling are not visible.</ab>
+                                </layout>
+                                
+                                <layout columns="1" writtenLines="33">
+                                    <note>Number of characters per line: 10–13.</note>
+                                    <note>The outer margins are delimited by a black line.</note>
+                                    <dimensions unit="mm" xml:id="textarea3">
+                                        <height>229</height>
+                                        <width>60–65</width>
+                                    </dimensions>
+                                    <dimensions type="margin" unit="mm" xml:id="margin3">
+                                        <dim type="right">5–15</dim>
+                                        <dim type="left">5–15</dim>
+                                    </dimensions>
+                                    <ab type="pricking">Pricking and ruling are not visible.</ab>
+                                </layout>
+                            </layoutDesc>
                             
                         </objectDesc>
                         
                         <handDesc>
                             <handNote xml:id="h1" script="Ethiopic">
-                           
+                           <desc>Modern bulky script, somewhat irregular. The nine-dotted asterisk is used often as word separator, and the word separator is often omitted when the end of a word 
+                               coincides with the end of the line.</desc>
+                                <date notBefore="1800" notAfter="2006"/>
```

Comment:
It was because the price is given in francs, but you are right, I didn't explain this. I've modified the formulation, and updated the date in palaeography.
On second thoughts, theoretically Swiss francs could also be meant...